bundle/manifests/backstage-default-config_v1_configmap.yaml

```diff
@@ -270,6 +284,8 @@ data:
               volumeMounts:
                 - mountPath: /opt/app-root/src/dynamic-plugins-root
                   name: dynamic-plugins-root
+                - mountPath: /var/log/audit
```

Comment:
looking at https://github.com/janus-idp/backstage-showcase/blob/7880332d0e0919dcbc08a9dd040392e146d9f05f/packages/backend/src/logger/customLogger.ts#L70 the default path for the audit log 

Is the plan to guide the user via documentation to set the correct path for this volume to be used?
@schultzp2020 @Zaperex

Reply:
I thought we had agreed upon /var/log/audit so not sure why it is different.